src/components/button/Button.tsx

```diff
@@ -51,6 +52,8 @@ const Button: React.FC<PButton> = ({
         size,
         variant,
         onClick,
+        role: 'button',
+        'aria-label': ariaLabel || label,
```

Comment:
[Aria guidelines](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) indicate to only use an `aria-label` if there isn't already a visible label provided. If we already have a label defined, then there's no need for an `aria-label` if one isn't specifically provided.